launch/tier4_control_launch/launch/control.launch.py

```diff
@@ -266,6 +297,7 @@ def launch_setup(context, *args, **kwargs):
             shift_decider_component,
             vehicle_cmd_gate_component,
             operation_mode_transition_manager_component,
+            obstacle_collision_checker_component,
```

Comment:
@TakaHoribe @yukkysaito How should we handle this? Is it okay for TIER IV to add this here, or should we consider another way?